src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityTreeCellRenderer.kt

```diff
@@ -97,6 +99,18 @@ class VulnerabilityTreeCellRenderer : ColoredTreeCellRenderer() {
                     text += obsoleteSuffix
                 }
             }
+            is ContainerImageTreeNode -> {
+                val issuesForImage = value.userObject as ContainerIssuesForImage
+                nodeIcon = SnykIcons.CONTAINER_IMAGE
+                text = issuesForImage.imageName
+
+                val toolWindowPanel = getSnykToolWindowPanel(value.project)
+                if (toolWindowPanel?.currentContainerResult == null) {
+                    attributes = SimpleTextAttributes.GRAYED_ATTRIBUTES
+                    nodeIcon = getDisabledIcon(nodeIcon)
+                    text += obsoleteSuffix
```

Comment:
Shouldn't the issue be set to obsolete, too, like in IaC?

Reply:
yes in next ticket: ROAD-451 detail panel for found vulnerabilities